frontend/templates/product/sections/ProductOverviewSection/ProductDescription.tsx

```diff
@@ -95,27 +96,12 @@ type VariantDescriptionProps = {
 
 function VariantDescription({ children }: VariantDescriptionProps) {
    return (
-      <Box
-         dangerouslySetInnerHTML={{
-            __html: children,
-         }}
+      <PrerenderedHTML
+         html={children}
          fontSize="sm"
          sx={{
-            ul: {
-               my: 3,
-               pl: 5,
-            },
-            p: {
+            'p:not(:last-of-type)': {
                mb: 3,
-               _last: {
-                  mb: 0,
-               },
-            },
```

Comment:
I'm slightly suspicious of this, because it looks like the `PrerenderedHTML` styles already include `p:not(:first-of-type)` for a similar thing. Do we want this, and does it interact fine with that? Is margin collapsing responsible for it looking okay?

Reply:
This raises an issue that's been heavy on my mind. InkofPixel has been moving towards a standardization effort starting in their designs where they avoid using top margin. Obviously, this is no small effort to convert, but I do see the value. This may be a good as place as any to start adoption.

I need to do some testing and get some time booked with Mattia to find out the motivator. I don't want to stall progress here, so I may just address the glaring issues for now.

Reply:
reverted differing styles in [`6eda824`](https://github.com/iFixit/react-commerce/pull/1946/commits/6eda824d4e1efde6fd5d96129dc768329aa242fd)